docs/cluster-readiness.md

```
@@ -0,0 +1,204 @@
+
+   ```json
+   {
+     "insecure-registries": ["registry.local", "10.96.0.0/12"]
+   }
+   ```
+
+   Apply & Restart. Docker Desktop's embedded k8s shares the Docker daemon's registry config.
+
+2. `/etc/hosts`:
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)